Auth0Tests/CredentialsManagerSpec.swift

```diff
@@ -454,8 +477,99 @@ class CredentialsManagerSpec: QuickSpec {
+                            done()
+                        }
+                    }
+                }
+
+                it("should yield a new access token with a new scope") {
+                    credentials = Credentials(accessToken: AccessToken, refreshToken: RefreshToken, expiresIn: Date(timeIntervalSinceNow: ExpiresIn), scope: "openid profile")
+                    _ = credentialsManager.store(credentials: credentials)
+                    waitUntil(timeout: 2) { done in
+                        credentialsManager.credentials(withScope: "openid profile offline_access", minTTL: 0) { error, newCredentials in
+                            expect(error).to(beNil())
+                            expect(newCredentials?.accessToken) == NewAccessToken
+                            done()
+                        }
+                    }
+                }
+
+                it("should not yield a new access token with a min ttl less than its expiry") {
+                    credentials = Credentials(accessToken: AccessToken, refreshToken: RefreshToken, expiresIn: Date(timeIntervalSinceNow: ExpiresIn))
+                    _ = credentialsManager.store(credentials: credentials)
+                    waitUntil(timeout: 2) { done in
+                        let ttl = Int((ExpiresIn - 1000) / 1000)
+                        credentialsManager.credentials(withScope: nil, minTTL: ttl) { error, newCredentials in
+                            expect(error).to(beNil())
+                            expect(newCredentials?.accessToken) == AccessToken
+                            done()
+                        }
+                    }
+                }
+
+                it("should yield a new access token with a min ttl greater than its expiry") {
+                    credentials = Credentials(accessToken: AccessToken, refreshToken: RefreshToken, expiresIn: Date(timeIntervalSinceNow: ExpiresIn))
+                    _ = credentialsManager.store(credentials: credentials)
+                    waitUntil(timeout: 2) { done in
+                        let ttl = Int((ExpiresIn + 1000) / 1000)
+                        credentialsManager.credentials(withScope: nil, minTTL: ttl) { error, newCredentials in
+                            expect(error).to(beNil())
+                            expect(newCredentials?.accessToken) == NewAccessToken
+                            done()
+                        }
+                    }
+                }
+
+                it("should fail to yield a renewed access token with a min ttl greater than its expiry") {
+                    credentials = Credentials(accessToken: AccessToken, refreshToken: RefreshToken, expiresIn: Date(timeIntervalSinceNow: ExpiresIn))
+                    _ = credentialsManager.store(credentials: credentials)
+                    waitUntil(timeout: 2) { done in
+                        credentialsManager.credentials(withScope: nil, minTTL: 100) { error, newCredentials in
```

Comment:
since `minTtl` for this scenario should be greater than `ExpiresIn`, let's reference this variable and add something to it instead of hardcoding the `100`.

Reply:
I think it's acceptable in a one-off unlike `ExpiresIn`, which is used everywhere in this file.

Reply:
Much like code, okay if it's a one-off use. If this 100 value is showing up more than once it should be a const. I read this as a suggestion vs a required change.